### PR TITLE
Color required module names in gray

### PR DESCRIPTION
**Pull request by ixnoahlive**
I made this a new pull since the original code was very far gone. This was tested & works. If anyone has issues running it please let me know and I'll look into it.

### CODE REVIEW

**Comment by ixnoahlive** [comment]
Bump

**Comment by ixnoahlive** [comment]
looks good to merge in my opinion

File: src/main/kotlin/com/chattriggers/ctjs/engine/module/Module.kt [hunk 1]
```diff
@@ -25,8 +25,13 @@ class Module(val name: String, var metadata: ModuleMetadata, val folder: File) {
             0xaa000000,
             x, y, width, 13f
         )
+
         Renderer.drawStringWithShadow(
-            metadata.name ?: name,
+            ChatLib.addColor(if (metadata.isRequired && requiredBy.isNotEmpty()) {
+                "&7"+metadata.name ?: name
```
Comment (mattco98) [inline src/main/kotlin/com/chattriggers/ctjs/engine/module/Module.kt]:
```suggestion
                "&7{metadata.name ?: name}"
```

**Review by mattco98 — CHANGES_REQUESTED** [review]

**Comment by ixnoahlive** [comment]
woops, had that in my compiled ver but never committed it! good catch!

**Comment by ixnoahlive** [comment]
squashed into 1 commit, lgtm

**Pull request merged**